Assets/Scripts/Scoreboard/Scoreboard.cs

```diff
@@ -229,4 +229,15 @@ private string FormatScore(int score)
             return $"{score}/{MatchRules.Current.MatchWinCondition.AmountForStopCondition}";
         return $"{score}";
     }
+
+    public void SetUIMask(bool isMasking)
+    {
+        // UI or default
+        var layer = isMasking ? 5 : 0;
+        gameObject.layer = layer;
+        foreach (Transform child in transform)
+        {
+            child.gameObject.layer = layer;
```

Comment:
Not recursive?

Reply:
Don't actually need to because the outline shader pass creates a mask that already covers the relevant parts of the screen 🤠